server/rest/src/test/java/org/infinispan/rest/search/aggregation/RestQueryAggregationCountTest.java

```diff
@@ -128,9 +128,9 @@ public void test() throws Exception {
    }
 
    private void writeEntries(RestCacheClient cacheClient) {
-      List<CompletionStage<RestResponse>> responses = new ArrayList<>(CHUNK_SIZE * NUMBER_OF_DAYS);
       for (int day = 1; day <= NUMBER_OF_DAYS; day++) {
          HashMap<String, Sale> chunk = chunk(day, fixedSeedPseudoRandom);
+         List<CompletionStage<RestResponse>> responses = new ArrayList<>(CHUNK_SIZE);
```

Comment:
Can't we just make this sync ?

Reply:
In this implementation we sync 1 chunk (10 items) at the time. The system seems to address this well. I don't know if it is the case to sync on each item.